go/integTest/shared_commands_test.go

```diff
@@ -845,7 +845,7 @@ func (suite *GlideTestSuite) TestHVals_WithNotExistingKey() {
 
 		res, err := client.HVals(key)
 		assert.Nil(suite.T(), err)
-		assert.Nil(suite.T(), res)
+		assert.Equal(suite.T(), []api.Result[string]{}, res)
```

Comment:
Why are you changing API for already existing commands?
If it is reasonable - update docs for them

Reply:
The expectation for `HVals` and `HKeys` command is to `return an empty slice when key does not exist.` and the doc comments for these commands already reflect the same whereas the function was returning `nil` for a non-existent key. Hence, I am only fixing the functionality and tests here.